js/api.md

```diff
@@ -456,33 +381,112 @@ Also, you can use `subscription` and `onSubscriptionMsg` attributes to enable su
 
 ```js
 
-<Connect  query={graphqlOperation(GetEvent, { id: currEventId })}
-          subscription={graphqlOperation(SubscribeToEventComments, { eventId: currEventId })}
-          onSubscriptionMsg={(prev, { subscribeToEventComments }) => {
-            console.log ( subscribeToEventComments);
-            return prev; }}>
-    {({ data: { listEvents } }) => (
-        <AllEvents events={listEvents ? listEvents.items : []} />
-    )}
+<Connect
+    query={graphqlOperation(queries.listTodos)}
+    subscription={graphqlOperation(subscriptions.onCreateTodo)}
+    onSubscriptionMsg={(prev, { onCreateTodo }) => {
+        console.log ( onCreateTodo );
+        return prev; 
+    }}
+>
+    {({ data: { listTodos }, loading, error }) => {
+        if (error) return (<h3>Error</h3>);
+        if (loading || !listTodos) return (<h3>Loading...</h3>);
+        <ListView todos={listTodos ? listTodos.items : []} />
+    }
  </Connect>
 
 ```
 
 For mutations, a `mutation` function needs to be provided with `Connect` component. `mutation` returns a promise that resolves with the result of the GraphQL mutation.
```

Comment:
update to:

For mutations, a `mutation` function needs to be provided with the `Connect` component. A `mutation` returns a promise that resolves with the result of the GraphQL mutation.